areas/storage/src/AzureMcp.Storage/StorageSetup.cs

```diff
@@ -88,6 +88,7 @@ access storage resources accessible to the authenticated user.
         blobs.AddCommand("list", new BlobListCommand(loggerFactory.CreateLogger<BlobListCommand>()));
 
         batch.AddCommand("set-tier", new BatchSetTierCommand(loggerFactory.CreateLogger<BatchSetTierCommand>()));
```

Comment:
This isn't related to this PR, but please file an issue to change this to "tier set" instead of set-tier.  Thanks